composer/trainer/trainer.py

```diff
@@ -1912,10 +1912,17 @@ def _train_loop(self) -> None:
                     self.engine.run_event(Event.AFTER_DATALOADER)
 
                     self.engine.run_event(Event.BATCH_START)
+
+                    # Log time values
                     self.logger.log_metrics({
-                        'trainer/global_step': int(self.state.timestamp.batch),
-                        'trainer/batch_idx': self.state.timestamp.batch_in_epoch.value,
+                        'time/batch': self.state.timestamp.batch.value,
+                        'time/sample': self.state.timestamp.sample.value,
+                        'time/batch_in_epoch': self.state.timestamp.batch_in_epoch.value,
```

Comment:
decided to keep the "in epoch" ones?

Reply:
I realized batch_in_epoch is pretty important for vision runs